### PR TITLE
[IMP] website: remove progress bar title margin

### PR DESCRIPTION
**Pull request by younn-o**
The mb0 has been added to the h4 tag of the progress bar so that there
is not too much padding if it is changed to simple text.

task-2542318

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website/views/snippets/s_media_list.xml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <odoo>
 
 <template name="Media List" id="s_media_list">
-    <section class="s_media_list bg-200 pt32 pb32" data-vcss="001">
+    <section class="s_media_list pt32 pb32 o_cc o_cc2" data-vcss="001">
```
Comment (qsm-odoo) [inline addons/website/views/snippets/s_media_list.xml:5]: This needs a theme PR, some themes override all the snippets you modify to change that bg-200 into something else, you have to adapt those.

**Review by qsm-odoo — CHANGES_REQUESTED** [review]

File: addons/website/views/snippets/s_media_list.xml (hunk 1 above)
Comment (younn-o) [reply addons/website/views/snippets/s_media_list.xml:5]: I have pushed a new PR here https://github.com/odoo/design-themes/pull/15

**Review by qsm-odoo — APPROVED** [review]
@robodoo rebase-ff r+

**Comment by qsm-odoo** [comment]
@robodoo r+

**Comment by qsm-odoo** [comment]
@robodoo r+

**Pull request closed without merging**